tests/splitter/test_washover.py

```diff
@@ -139,3 +140,44 @@ class Cfg:
         assert not washover_df.query("time >= '2022-01-01 01:00:00'").equals(
             out_df.query("time >= '2022-01-01 01:00:00'")
         )
+
+
+def test_truncated_time_not_in_cluster_cols():
+    msg = "is not in the cluster columns."
+    with pytest.raises(ValueError, match=msg):
+        df = pd.DataFrame(columns=["time_bin", "city", "time", "treatment"])
```

Comment:
I'd create the dataframe before the with statement, I'd also add some comment on what the test tries to capture